src/github.ts

```diff
@@ -82,7 +82,7 @@ export const createBackportPr = async (
         )
       }`,
       base: `release/v${giteaVersion.majorMinorVersion}`,
-      body: `Backport #${originalPr.number}\n\n` + originalPr.body,
+      body: `Backport #${originalPr.number} by @${originalPr.user.login}\n\n${originalPr.body}`,
```

Comment:
```suggestion
      body:
        `Backport #${originalPr.number} by @${originalPr.user.login}\n\n${originalPr.body}`,
```
Diff from my `deno fmt`